usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -37,8 +37,53 @@ blacklist {
     modprobe dm-multipath >&2
     multipath >&2
     if [ $? -ne 0 ] ; then
+        # Asking to the User what to do after multipath command return 1.
+        # It could be because no multipath device were found (sles11/rhel6)
+        # or a real problem in the multipath configuration.
         LogPrint "Failed to activate multipath, or no multipath device found."
-        rear_shell "Did you activate the multipath devices?"
+
+        rear_workflow="rear $WORKFLOW"
+        unset choices
+        choices[0]="Multipath is not needed, please continue recovery."
+        choices[1]="Run multipath with debug options."
+        choices[2]="Enter into rear-shell to manually debug multipath."
+        choices[3]="Abort '$rear_workflow'"
+        prompt="Choice:"
+        choice=""
+        wilful_input=""
+
+        # looping on this menu while multipath failed to list device.
+        # Note: On sles11/rhel6, multipath failed if no multipath device is found.
+        while ! multipath ; do
+            echo
+            choice="$( UserInput -t 30 -p "$prompt" -D "${choices[0]}" "${choices[@]}")"&& wilful_input="yes" || wilful_input="no"
```

Comment:
If I understand the `UserInput` usage correctly then `-D 0` should also work and avoid the repetition of the choice text.